Hexastack/Hexabot
# feat: add llm helper + Ollama

# Motivation

In this PR : 
- We add LLM helper and Ollama as the default
- We add docker compose files for Ollama service
- We add a RAG generative AI plugin that works with Ollama

# How to test
- In settings a new Ollama should be available and settings form is translated and has helper text
- Run the project using "npx hexabot dev --enable=ollama", ssh into the container and pull tinyllama "ollama pull tinyllama"
- Drag and drop a Ollama block in the visual editor and test

# Type of change:

Please delete options that are not relevant.


# Checklist:
